docker/database/init-db.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+ set -e
+
+ if [ -z "$JUDICIAL_PAYMENT_SERVICE_DB_USERNAME" ] || [ -z "$JUDICIAL_PAYMENT_SERVICE_DB_PASSWORD" ]; then
+   echo "ERROR: Missing environment variable. Set value for both 'JUDICIAL_PAYMENT_SERVICE_DB_USERNAME' and 'JUDICIAL_PAYMENT_SERVICE_DB_PASSWORD'."
+   exit 1
+ fi
+
+ # Create role and database
+ psql -v ON_ERROR_STOP=1 --username postgres --set USERNAME=$JUDICIAL_PAYMENT_SERVICE_DB_USERNAME --set PASSWORD=$JUDICIAL_PAYMENT_SERVICE_DB_PASSWORD <<-EOSQL
+   CREATE USER :USERNAME WITH PASSWORD ':PASSWORD';
+   CREATE DATABASE hmc_judicial-payment-service
```

Comment:
```suggestion
   CREATE DATABASE hmc_judicial_payment_service
```

Reply:
I also think we need to declare these values in a env file JUDICIAL_PAYMENT_SERVICE_DB_USERNAME and JUDICIAL_PAYMENT_SERVICE_DB_PASSWORD